assets/index.html

```diff
@@ -10,6 +10,13 @@
     />
     <script src="vendors/bootstrap-5.2.3/js/bootstrap.bundle.min.js"></script>
     <!-- bootstrap end -->
+    <!-- custom css start -->
+    <link
+    rel="stylesheet"
+    type="text/css"
+    href="css/customize.css"
+    />
```

Comment:
you can uncomment this file and add a documentation to indicate if you wanna some customization of the css, uncomment following line.